Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -91,6 +91,7 @@ public typealias StartPurchaseBlock = (@escaping PurchaseCompletedBlock) -> Void
             // Sends cached customer info (if exists) to delegate as latest
             // customer info may have already been observed and sent by the monitor
             self.sendCachedCustomerInfoToDelegateIfExists()
+            // TODO: do we need to send it the first time when the dangerousSetting is active? 
```

Comment:
this is an interesting one. If you're doing your own entitlement computation, and would like to rely on the delegate as a proxy for knowing when you need to recompute, then firing right away after setting might not be useful. It might be more useful to only fire this one when an actual value comes in, which if everything else is disabled, would only happen when a post receipt goes through successfully